src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/components/ColorComponent/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { FormGroup, FormControlLabel, Checkbox, FormLabel } from '@mui/material';
+import styles from './Color.module.scss';
+
+const colorOptions = [
+  'Black',
+  'Blue',
+  'Brown',
+  'Gold',
+  'Gray',
+  'Green',
+  'Maroon',
+  'Orange',
+  'Pink',
+  'Purple',
+  'Red',
+  'Silver',
+  'Tan',
+  'White',
+  'Yellow',
+];
+
+type ColorProps = {
+  selectedColors: string[];
+  onColorChange: (colors: string[]) => void;
+};
+
+const Color: React.FC<ColorProps> = ({ selectedColors = [], onColorChange }) => {
+  const handleColorToggle = (color: string) => {
+    const updatedColors = selectedColors.includes(color)
+      ? selectedColors.filter((c) => c !== color)
+      : [...selectedColors, color];
+    onColorChange(updatedColors);
+  };
+
+  return (
+    <div className={styles.color_group}>
+      <FormLabel>
+        Item Color: Choose <u>ALL</u> that apply:
+      </FormLabel>
+      <FormGroup className={styles.checkbox_group}>
+        {colorOptions.map((color) => (
+          <FormControlLabel
+            key={color}
+            control={
+              <Checkbox
+                checked={selectedColors.includes(color)}
+                onChange={() => handleColorToggle(color)}
+              />
+            }
+            label={color}
+          />
+        ))}
+      </FormGroup>
+    </div>
+  );
+};
+
+export default Color;
```

Comment:
Is this component being used anywhere?

Reply:
Currently no, I'm still working on how to incorporate that, i might remove it in my next iteration of what i work on if im still not able to figure that out

Reply:
If it's unused, it should be removed from this PR before we merge, you can keep it locally if it's in progress, but I don't think we shouldn't merge it to our shared branch.

Reply:
sounds good

Reply:
> @richmondnao If it's unused, it should probably be removed from this PR before we merge, you can keep it in a branch locally, or in a separate PR if it's in progress, but I don't think we shouldn't merge it to our shared branch.

removed unused components